prisma/robochimp.prisma

```diff
@@ -28,5 +28,10 @@ model User {
     id   BigInt @id @unique
     bits Int[]
 
+    leagues_completed_tasks_ids Int[]
```

Comment:
This will likely need something similar to the startupScripts.ts to set a default value and not null.

I know its robochimp but still probably a good idea.